### ISSUE TITLE
Avoid using bundlephobia

### ISSUE THREAD

**Issue by AbhiPrasad** [npmgraph/npmgraph#220]
Bundlephobia doesn't take into account multiple entry points and export conditions, which means displays the wrong info.

Something like https://pkg-size.dev/ is more accurate, but anything that takes into accounts entry points works better.

**Comment by broofa** [comment]
@styfle and I have had [extended conversation](https://github.com/npmgraph/npmgraph/issues/132) about the meaning "bundle size".  tl;dr: There isn't really one "right" answer... not unless you know the entry points being called and run a case-specific build with tree-shaking and whatnot.  While pkg-size has an option for that (it allows you to specify which exports should be considered entry points), that's a pretty onerous "ask" of users.   And without that, it's not all that different from bundlephobia.

tl;dr: npmgraph's bundle size #'s are known to be imperfect... and I'm okay with that for now.

**Comment by AbhiPrasad** [comment]
That feels reasonable - but I wonder if there's a better way to highlight this to users.

Many don't know about the drawbacks of bundlephobia and miss out on possible bundle size savings, and given more and more modern JavaScript packages are going to use export conditions this problem will only grow larger.

Thank you for the quick response!
